micro-infra-spring-base/src/main/groovy/com/ofg/infrastructure/web/resttemplate/custom/ResponseRethrowingErrorHandler.groovy

```diff
@@ -29,7 +52,15 @@ class ResponseRethrowingErrorHandler implements ResponseErrorHandler {
 
     protected String getLoggedErrorResponseBody(ClientHttpResponse response) {
         String responseBody = getResponseBody(response)
-        log.error("Response error: status code [$response.statusCode], headers [$response.headers], body [$responseBody]")
+
+        switch (errorMessageLogging) {
+            case ErrorMessageLogging.ERROR:
+                log.error("Response error: status code [$response.statusCode], headers [$response.headers], body [$responseBody]")
+                break
+            case ErrorMessageLogging.WARN:
+                log.warn("Response error: status code [$response.statusCode], headers [$response.headers], body [$responseBody]")
+                break
```

Comment:
what do you mean by `move to separate method`, move whole switch to method like `logErrorResponse` (if so, then I'd still need default clause and comment), or create path `default: doNothing()`?